images/win/toolsets/toolset-2016.json

```diff
@@ -121,7 +121,7 @@
             "name": "az",
             "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
             "versions": [
-                "5.5.0"
+                "5.7.0"
             ],
             "zip_versions": [
                 "1.0.0",
```

Comment:
As per telemetry data, there are ~10k jobs a month that have hardcoded 5.5.0
I would suggest to add 5.5.0 in zip_versions and remove 4.6.0 for both win versions.

![image](https://user-images.githubusercontent.com/8944724/113977050-9ee42900-985f-11eb-93d0-ab9430926d81.png)

Reply:
@amit-avit thanks for you suggestion, added!